src/index.js

```diff
@@ -59,14 +58,59 @@ function cleanupAtContainer(container) {
   mountedContainers.delete(container)
 }
 
+const originalChange = fireEvent.change
```

Comment:
Is it necessary to override the `change` event for this to work? I think we can remove the `change` event code here... I think that one works as-is

Reply:
The same as above, but I can check if it's necessary.